gui/src/main.rs

```diff
@@ -279,11 +317,13 @@ impl Application for GUI {
                 State::Launcher(v) => v.subscription().map(|msg| Message::Launch(Box::new(msg))),
             },
             iced_native::subscription::events().map(Self::Message::Event),
+            self.key_subscription(),
```

Comment:
self.key_subscription is using iced_native::subscriptions::events_with, which is the same subscription that the one line 319. Please merge them together like this:

```rust
          iced_native::subscription::events_with(|event, status| match (&event, status) {
                 (
                    Event::Keyboard(keyboard::Event::KeyPressed {
                        key_code: KeyCode::Tab,
                        modifiers,
                        ..
                    }),
                    event::Status::Ignored,
                ) => Some(Message::KeyPressed(Key::Tab(modifiers.shift()))),
                (
                    iced::Event::Window(iced_native::window::Event::CloseRequested),
                    event::Status::Ignored,
                ) => Some(Message::Event(event)),
                _ => None,
            }),
```

Reply:
done